signatures/alert_signatures.py

```diff
@@ -0,0 +1,1012 @@
+    "ET POLICY Metasploit Framework Checking For Update": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET POLICY Dropbox DNS Lookup - Possible Offsite File Backup in Use": MicroAttackStage.DATA_EXFILTRATION,
+    "ET WEB_SERVER Possible IIS Integer Overflow DoS (CVE-2015-1635)": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET USER_AGENTS MSF Meterpreter Default User Agent": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET ATTACK_RESPONSE Metasploit Meterpreter Reverse HTTPS certificate": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET WEB_SERVER Possible CVE-2014-6271 Attempt": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET EXPLOIT Serialized Java Object Calling Common Collection Function": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET POLICY Possible HTA Application Download": MicroAttackStage.DATA_DELIVERY,
+    "ET CURRENT_EVENTS SUSPICIOUS Firesale gTLD EXE DL with no Referer June 13 2016": MicroAttackStage.DATA_DELIVERY,
+    "ET POLICY Possible Kali Linux hostname in DHCP Request Packet": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET WEB_CLIENT HTA File containing Wscript.Shell Call - Potential CVE-2017-0199": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET SCAN Possible Nmap User-Agent Observed": MicroAttackStage.HOST_DISC,
+    "ET POLICY Outdated Flash Version M2": MicroAttackStage.NON_MALICIOUS,
+    # "ET CURRENT_EVENTS Possible AMSI Powershell Bypass Attempt B642": MicroAttackStage.DEFENSE_EVASION,
+    "ET WEB_CLIENT PowerShell call in script 1": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET WEB_CLIENT PowerShell call in script 2": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET EXPLOIT Cisco Smart Install Exploitation Tool - ChangeConfig": MicroAttackStage.SERVICE_SPECIFIC,
+    "ET EXPLOIT Cisco Smart Install Exploitation Tool - GetConfig": MicroAttackStage.SERVICE_SPECIFIC,
+    "ET POLICY SMB Executable File Transfer": MicroAttackStage.DATA_EXFILTRATION,
+    "ET POLICY SMB2 NT Create AndX Request For an Executable File": MicroAttackStage.DATA_DELIVERY,
+    "ET POLICY SMB2 NT Create AndX Request For a Powershell .ps1 File": MicroAttackStage.DATA_DELIVERY,
+    "ET POLICY SMB2 NT Create AndX Request For a .bat File": MicroAttackStage.DATA_DELIVERY,
+    "ET POLICY SMB2 NT Create AndX Request For a DLL File - Possible Lateral Movement": MicroAttackStage.DATA_DELIVERY,
+    "ET POLICY SMB2 Remote AT Scheduled Job Create Request": MicroAttackStage.DATA_EXFILTRATION,
+    "ET POLICY SMB Remote AT Scheduled Job Pipe Creation": MicroAttackStage.DATA_EXFILTRATION,
+    "ET POLICY Powershell Activity Over SMB - Likely Lateral Movement": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET POLICY Powershell Command With Hidden Window Argument Over SMB - Likely Lateral Movement": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET POLICY Powershell Command With No Profile Argument Over SMB - Likely Lateral Movement": MicroAttackStage.COMMAND_AND_CONTROL,
+    "GPL CHAT Jabber/Google Talk Outgoing Traffic": MicroAttackStage.INFO_DISC,
```

Comment:
This alert signature ("GPL CHAT Jabber/Google Talk Outgoing Traffic") is `MicroAttackStage.INFO_DISC`, whereas below it is `MicroAttackStage.NON_MALICIOUS`. Which one to keep?

Reply:
From a quick Google search, it seems to be originating from printers or other devices with Google Talk enabled. I'd say: NON_MALICIOUS